src/components/User.js

```diff
@@ -0,0 +1,100 @@
+import firebaseConfig from "../firebase";
+import { getDatabase, onValue, ref, update, remove, runTransaction } from "firebase/database";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+const User = () => {
+  //Show all the thoughts that have been posted for the uid
+  const { userId } = useParams();
+  const [userThought, setUserThought] = useState([]);
+  const database = getDatabase(firebaseConfig);
+  const dbRef = ref(database);
+
+  useEffect(() => {
```

Comment:
there is the same chunk of code as in the StoredThoughts.js component - maybe try to declare a variable that you can reuse instead?